compose/material/material/build.gradle

```diff
@@ -98,11 +99,11 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 implementation(libs.kotlinStdlib)
                 api(project(":compose:animation:animation-core"))
                 api(project(":compose:foundation:foundation"))
-                api("org.jetbrains.compose.material:material-icons-core:1.6.11") {
-                    // exclude dependencies, because they override local projects when we build 0.0.0-* version
-                    // (see https://repo1.maven.org/maven2/org/jetbrains/compose/material/material-icons-core-desktop/1.6.11/material-icons-core-desktop-1.6.11.module)
-                    exclude group: "org.jetbrains.compose.ui"
-                }
+//                api("org.jetbrains.compose.material:material-icons-core:1.6.11") {
+//                    // exclude dependencies, because they override local projects when we build 0.0.0-* version
+//                    // (see https://repo1.maven.org/maven2/org/jetbrains/compose/material/material-icons-core-desktop/1.6.11/material-icons-core-desktop-1.6.11.module)
+//                    exclude group: "org.jetbrains.compose.ui"
+//                }
```

Comment:
The `material-icons-core` module is unavailable for native Linux. What to do about this? Without the source code `material-icons-*` can never be supported. Which is probably fine, but in that case it needs to be removed as dependency here.

![Screenshot 2025-04-17 at 01 01 41](https://github.com/user-attachments/assets/d026f720-b1fa-441d-a049-7f24904a87d8)
https://kotlinlang.slack.com/archives/CJLTWPH7S/p1744660966776609?thread_ts=1743758275.030769&cid=CJLTWPH7S

Reply:
No longer blocking issue as material/material3 does not depend on it anymore. Publishing for icon modules for Linux can be skipped.